### PR TITLE
fix: Change placeholder color to muted and make popup appear directly

### PR DESCRIPTION
**Pull request by EDsCODE**
## Problem

- follow up to #10185 
- Make the placeholder text muted
- Make the selector popup by default when selecting
## Changes

https://user-images.githubusercontent.com/13127476/174861015-e8d4c0de-efb1-4812-af5b-9f95eb384b50.mov

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

**Review by clarkus — CHANGES_REQUESTED** [review]
Thanks for working on this. One thing I see that could be improved is a focus / active status for the input that corresponds to the taxonomic filter. Usually this is a blue highlight border. It'd reinforce which step is being modified and where the selected value will append when selected.

**Comment by EDsCODE** [comment]
Active state on the selector like this? Just added
Screen Shot 2022-06-21 at 1 49 35 PM

**Comment by clarkus** [comment]
Not quite - I mean the select element that contains / will contain the selected event. Typically the thing being affected gets a blue highlight border to indicate that it's active / being affected by whatever input actions are happening. In this case we're highlighting that event input to clarify that when you select an event, it'll populate there. Just helps to clarify which step is being modified. Figma example for focused / active inputs at https://www.figma.com/file/Y9G24U4r04nEjIDGIEGuKI/PostHog-Design-System-One?node-id=2395%3A4583

Screen Shot 2022-06-21 at 10 54 17 AM

**Comment by EDsCODE** [comment]
I think we may be talking about the same thing but the highlight is just super subtle and I wrote a bad descriptor. This is before and after

Before:
Screen Shot 2022-06-21 at 2 07 56 PM

After:
Screen Shot 2022-06-21 at 2 08 09 PM

This should be bolder right

**Comment by clarkus** [comment]
> This should be bolder right

Yeah maybe it's a result of using ant component, but the border should be at least 1px to contrast sufficiently. This could also be tied to the highlight color we're using. We have an outstanding task to change these but it hasn't been scheduled for work yet. Swapping the primary blue would definitely improve the contrast there. https://github.com/PostHog/posthog/issues/9868

**Comment by EDsCODE** [comment]
mmm, what you're seeing there is actually 1px. I just pushed it to 1.5px when highlighted so it's more clear. I have the color hooked onto the primary-hover color we define so when that color change is done it will apply here

Screen Shot 2022-06-21 at 2 29 34 PM

**Comment by clarkus** [comment]
> mmm, what you're seeing there is actually 1px. I just pushed it to 1.5px when highlighted so it's more clear. I have the color hooked onto the primary-hover color we define so when that color change is done it will apply here

I think that's fine short-term. Longer term I feel like we should standardize these inputs so they're consistent across features. We have a good mix of lemon and ant now, but eventually it'd be good to keep investing in lemon variants so we can have a consistent experience.

**Review by clarkus — APPROVED** [review]

**Review by liyiy — APPROVED** [review]

File: frontend/src/lib/components/EntityFilterInfo.tsx [hunk 1]
```diff
@@ -30,7 +30,11 @@ export function EntityFilterInfo({
 
     // No filter
     if (filter.type === EntityTypes.NEW_ENTITY || !title) {
-        return <TextWrapper title="Select filter">Select filter</TextWrapper>
+        return (
+            <TextWrapper title="Select event" style={{ color: 'var(--muted-alt)' }}>
+                Select event
```
Comment (liyiy) [inline frontend/src/lib/components/EntityFilterInfo.tsx:35]: this renaming is probably causing tests to fail?

**Pull request merged**